extensions/wikia/AdEngine/js/lookup/openXBidder.js

```diff
@@ -0,0 +1,178 @@
+/*global define*/
+/*jshint camelcase:false*/
+/*jshint maxdepth:5*/
+define('ext.wikia.adEngine.lookup.openXBidder', [
+	'ext.wikia.adEngine.adTracker',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (adTracker, doc, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.lookup.openXBidder',
+		oxResponse = false,
+		oxTiming,
+		bestPricePointForSize = {
+			'1x6': null,
+			'3x2': null,
+			'3x5': null,
+			'3x6': null,
+			'7x9': null
+		},
+		called = false,
+		slots = {
+			HOME_TOP_LEADERBOARD: '728x90',
+			HOME_TOP_RIGHT_BOXAD: '300x250',
+			INCONTENT_BOXAD_1: '300x250',
+			LEFT_SKYSCRAPER_2: '160x600',
+			LEFT_SKYSCRAPER_3: '160x600',
+			PREFOOTER_LEFT_BOXAD: '300x250',
+			PREFOOTER_RIGHT_BOXAD: '300x250',
+			TOP_LEADERBOARD: '728x90',
+			TOP_RIGHT_BOXAD: '300x250',
+			MOBILE_IN_CONTENT: '300x250',
+			MOBILE_PREFOOTER: '300x250',
+			MOBILE_TOP_LEADERBOARD: '320x50'
+		},
+		sizeMapping = {
+			'728x90': '7x9',
+			'300x250': '3x2',
+			'160x600': '1x6',
+			'320x50': '3x5'
+		};
+
+	function getAds() {
+		var ads = [],
+			size,
+			slotName,
+			slotPath,
+			src = 'gpt';
+
+		for (slotName in slots) {
+			if (slots.hasOwnProperty(slotName)) {
+				size = slots[slotName];
+				slotPath = [
+					'/5441', 'wka.life', '_prowrestling', 'article', src, slotName
```

Comment:
We were talking about it yesterday on daily: let's mark it with `TODO`, so in the upcoming work we'll remove the hardcoded "wiki name".